lib/sles4sap_publiccloud.pm

```diff
@@ -335,18 +335,28 @@ sub stop_hana {
     if ($args{method} eq "crash") {
         # Crash needs to be executed as root and wait for host reboot
         $self->{my_instance}->wait_for_ssh(timeout => $timeout);
```

Comment:
If I can add a kind suggestion, this call is not clear here if is waiting for some starting system; but in the case the ssh sut is wellknown to be already up in this point, the wait call could even be removed.